ibis/backends/tests/test_param.py

```diff
@@ -116,21 +121,42 @@ def test_scalar_param_map(con):
             "timestamp",
             "timestamp_col",
             id="string_timestamp",
-            marks=pytest.mark.notimpl(["druid"]),
+            marks=[
+                pytest.mark.notimpl(["druid"]),
+                pytest.mark.broken(
+                    ["bigquery"],
+                    raises=GoogleBadRequest,
+                    reason="No matching for operator = for argument types: DATETIME, TIMESTAMP",
+                ),
+            ],
         ),
         param(
             datetime.date(2009, 1, 20),
             "timestamp",
             "timestamp_col",
             id="date_timestamp",
-            marks=pytest.mark.notimpl(["druid"]),
+            marks=[
+                pytest.mark.notimpl(["druid"]),
+                pytest.mark.broken(
+                    ["bigquery"],
+                    raises=GoogleBadRequest,
+                    reason="No matching for operator = for argument types: DATETIME, TIMESTAMP",
+                ),
+            ],
         ),
         param(
             datetime.datetime(2009, 1, 20, 1, 2, 3),
             "timestamp",
             "timestamp_col",
             id="datetime_timestamp",
-            marks=pytest.mark.notimpl(["druid"]),
+            marks=[
+                pytest.mark.notimpl(["druid"]),
+                pytest.mark.broken(
+                    ["bigquery"],
```

Comment:
We can make these pass, but we'll need to adjust the column type of `timestamp_col` in `functional_alltypes`. That can be done in a follow up.

Reply:
Yeah, I checked `functional_alltypes` is defined at `ibis/backends/conftest.py` for all backends. But I didn't find a way to specify `timestamp_col` as `timestamp` with timezone, and I was worried about breaking other backend so that I didn't make it. Feel free to let me know if I can help for follow up here.